elephant/asset.py

```diff
@@ -103,6 +103,7 @@
 import warnings
 
 import neo
+import math
```

Comment:
You don't seem to be using this unless I'm missing something. `itertools` should also be unused now.

Reply:
yes, I used math trying to come up with a closed-form solution for `_num_iterations`.
Removed both unused math and itertools.